tools/generator/Java.Interop.Tools.Generator.Transformation/JavadocFixups.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Xml.Linq;
+
+using Java.Interop.Tools.JavaSource;
+
+using MonoDroid.Generation;
+using MonoDroid.Utils;
+
+using Xamarin.Android.Binder;
+
+namespace Java.Interop.Tools.Generator.Transformation
+{
+	public static class JavadocFixups
+	{
+		public static void Fixup (List<GenBase> gens, CodeGeneratorOptions options)
+		{
+			if (options.JavadocXmlFiles == null || options.JavadocXmlFiles.Count == 0)
+				return;
+			foreach (var path in options.JavadocXmlFiles) {
+				if (!File.Exists (path))
```

Comment:
Should this be a warning or message to the user?  It feels like if they passed in a file they are expecting us to use it, so we should let them know if we aren't able to.

Reply:
This has been reworked to raise warning `BG8600` if it doesn't exist or is otherwise not a valid XML file.